### DISCUSSION TITLE
shouldn't Package.resolved be added to .gitignore?

### DISCUSSION THREAD

**Discussion by byaruhaf** [mozilla-mobile/firefox-ios#11463] [Q&A] [answered]
Since this file is auto-generated every time dependency resolution is performed, is there a reason it should go in version control?

**Comment by lmarceau** [comment, marked as answer]
We keep it in version control since we want everyone to use the same set of dependencies. `Package.resolved` file records the result of dependency resolution whenever SwiftPM performs it. If the file is already present, SwiftPM will try to resolve to the same versions as recorded in the file.
